src/block.rs

```diff
@@ -16,10 +16,10 @@ pub type BlockResult<T> = core::result::Result<T, BlockError>;
 
 /// Represent a certain amount of data from a block device.
 #[derive(Clone)]
-#[repr(C, align(512))]
+#[repr(C)]
 pub struct Block {
     /// The actual storage of the block.
-    pub contents: [u8; Block::LEN],
+    pub contents: [u16; Block::LEN / 2],
```

Comment:
Lol... I meant do an u16 array in the .id (which I forgot wasn't possible). This should definitely be `repr(C, align(2))` if we're going to force alignment to 2 here.

Reply:
![957P24t](https://user-images.githubusercontent.com/6343527/62865808-b2985e80-bd0f-11e9-8a76-330c9e3af19e.gif)